packages/sdk/src/index.spec.ts

```diff
@@ -0,0 +1,19 @@
+import assert from "assert";
+import SDK from "./index";
+import {
+  PEER_ID_A as ORIGIN,
+  PROVIDER,
+  RPC_REQ_SMALL,
+} from "rpch-commons/src/fixtures";
```

Comment:
it doesn't look like fixtures are exported yet, so before we merge this PR we need to make sure `rpch/commons` is updated to export fixtures as this is not the right way to important them

Reply:
added export to commons 👍